server/controller/textScan.controller.js

```diff
@@ -48,6 +48,8 @@ textScanController.post(
         semesters: semesters
           .filter(semester => semester._id)
           .map(semester => semester._id),
+        remainingRequirement,
```

Comment:
The Plan schema also includes a field 'program' which contains an ID to a degree program. Shouldn't we also add a 'program' field here?

Reply:
I guess for now since we're only doing software engineering. We can just hard code it to "Software Engineering"?

Reply:
Yeah, for now, I don't do anything with Program Controller or create new Program, so I don't add it here as well.